trulens_eval/trulens_eval/tests/groundedness_smoke_tests.ipynb

```diff
@@ -0,0 +1,325 @@
+{
```

Comment:
<p>Thanks for changing the feedback var name to f_mae. </p><p> </p><p>To close the loop, I'd suggest 3 small changes:</p><ul><li>Change the human-readable name being passed from "Groundedness Smoke Test" to "Mean Absolute Error"</li><li>Change the implementation of numeric_difference <a href="            ret = 1 - abs(float(score) - expected_score)" target="_blank">here</a> to remove the score inversion</li><li>Merge in Jisoo's "higher is better" parameter (approved <a href="https://github.com/truera/trulens/pull/528" target="_blank">PR</a>) and then set this to False for this feedback definition.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I put in these changes - @daniel-huang-1230 can you give them a look and then we can merge?

Reply:
thank you Josh for taking care of those! lgtm